includes/entities/class-webmention-item.php

```diff
@@ -134,13 +137,16 @@ public function __call( $method, $params ) {
 	}
 
 	public function set( $key, $value, $overwrite = false ) {
+		if ( in_array( $key, array( 'raw', 'site_name', 'response_type' ), true ) ) {
+			$key = '_' . $key;
```

Comment:
this might cause errors if you miss adding new params. I would keep the extra underscore, to show the difference even on the setter and getter

Reply:
or we introduce something like `set_meta( 'key', 'value' )` to handle the underscores... If we add new params, we shouldn't add them to multiple locations, array, settings...

Reply:
perhaps we should not prefix the class attributes at all...